FSharp.Core.Nuget/FSharp.Core.nuspec

```diff
@@ -114,5 +114,9 @@
     <file src="..\packages\Microsoft.FSharp.Core.netcore.1.0.0-rc-170122\lib\netstandard1.6\FSharp.Core.optdata" target="lib\netstandard1.6\FSharp.Core.optdata" />
     <file src="..\packages\Microsoft.FSharp.Core.netcore.1.0.0-rc-170122\lib\netstandard1.6\FSharp.Core.sigdata" target="lib\netstandard1.6\FSharp.Core.sigdata" />
     <file src="..\packages\Microsoft.FSharp.Core.netcore.1.0.0-rc-170122\lib\netstandard1.6\FSharp.Core.xml" target="lib\netstandard1.6\FSharp.Core.xml" />
+    <file src="..\packages\Microsoft.FSharp.Core.netcore.1.0.0-rc-170122\lib\netstandard1.6\FSharp.Core.optdata" target="runtimes\any\native\FSharp.Core.optdata" />
+    <file src="..\packages\Microsoft.FSharp.Core.netcore.1.0.0-rc-170122\lib\netstandard1.6\FSharp.Core.sigdata" target="runtimes\any\native\FSharp.Core.sigdata" />
```

Comment:
When I look at ``Microsoft.FSharp.Core.netcore.1.0.0-rc-170122\runtimes\any\native`` I see the XML file as well:
```

30/03/2017  17:11    <DIR>          .
30/03/2017  17:11    <DIR>          ..
22/01/2017  18:19           430,090 FSharp.Core.optdata
22/01/2017  18:19           562,828 FSharp.Core.sigdata
22/01/2017  18:19           707,017 FSharp.Core.xml
               3 File(s)      1,699,935 bytes
               2 Dir(s)  26,637,254,656 bytes free

```